sample/HttpTrigger-Powershell/run.ps1

```diff
@@ -1,10 +1,8 @@
-if ($Env:req_query_name) 
-{
```

Comment:
I'd like to keep this existing sample as is, since it is designed to perform the same operations as the Node/C# examples. Recommend adding a new sample for your stuff. Perhaps "HttpTrigger-Powershell-Modules" or something